doc/release-notes.rst

```diff
@@ -2,6 +2,72 @@
  Release Notes
 ===============
 
+v0.87.2 Giant
+=============
+
+Notable Changes
+---------------
+* allow -L to disable lttng. Enable it by default (Alfredo Deza)
+* mon: MDSMonitor: wait for osdmon to be writable when requesting proposal (#9794 Joao Eduardo Luis)
+* mon: MDSMonitor: have management_command() returning int instead of bool (Joao Eduardo Luis)
+* osd: Get pgid ancestor from last_map when building past intervals (#10430 David Zafman)
+* Objecter: failed assert(tick_event==NULL) at osdc/Objecter.cc (#11183 Zhiqiang Wang)
+* ceph-objectstore-tool: Output only unsupported features when incomatible (#11176 David Zafman)
+* rgw: Swift API. Support for X-Remove-Container-Meta-{key} header. (#10475 Dmytro Iurchenko)
+* librados: Translate operation flags from C APIs (Matthew Richards)
+* Objecter: check the 'initialized' atomic_t safely (#9617 Josh Durgin)
+* Objecter: init with a constant of the correct type (Josh Durgin)
+* CrushWrapper: pick a ruleset same as rule_id (Xiaoxi Chen)
```

Comment:
It is http://tracker.ceph.com/issues/9675 according to https://github.com/ceph/ceph/pull/4095